vue/components/ui/tag-platforme/tag-platforme.vue

```diff
@@ -12,6 +12,7 @@
     display: inline-block;
     font-size: 12px;
     font-weight: 600;
+    height: auto;

```

Comment:
not a good idea let's force line-height instead and not auto a specific value like 14px